mne/io/ctf/ctf.py

```diff
@@ -182,8 +182,11 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                 samp_offset = (bi + trial_start_idx) * si['res4_nsamp']
                 n_read = min(si['n_samp_tot'] - samp_offset, si['block_size'])
                 # read the chunk of data
-                pos = CTF.HEADER_SIZE
-                pos += samp_offset * si['n_chan'] * 4
+                # have to be careful on Windows and make sure we are using
+                # 64-bit integers here
+                with np.errstate(over='raise'):
+                    pos = np.int64(CTF.HEADER_SIZE)
+                    pos += np.int64(samp_offset) * si['n_chan'] * 4
```

Comment:
that turned out to be less painful than I thought it would be

Reply:
... and it would have been even easier / not required a windows machine to debug at all if the native `int` emitted a warning when it overflowed like `numpy` does!